docs/upgrading.md

```diff
@@ -5,6 +5,10 @@ Below we describe the additional manual steps required to upgrade between differ
 
 # Open Autonomy
 
+## `v0.6.0` to `v0.7.0`
+
+No backwards incompatible changes
```

Comment:
Why a major release then?

Reply:
Sorry, I forgot about the comment. I think there were some breaking changes, I think either @DavidMinarsch or @Adamantios can help here

Reply:
See below. @Adamantios might be able to improve it further.